src/js/mep-feature-sourcechooser.js

```diff
@@ -13,27 +13,61 @@
 
 			player.sourcechooserButton =
 				$('<div class="mejs-button mejs-sourcechooser-button">'+
-					'<button type="button" aria-controls="' + t.id + '" title="' + t.options.sourcechooserText + '" aria-label="' + t.options.sourcechooserText + '"></button>'+
-					'<div class="mejs-sourcechooser-selector mejs-offscreen">'+
-						'<ul>'+
-						'</ul>'+
-					'</div>'+
-				'</div>')
+						'<button type="button" aria-haspopup="true" aria-owns="' + t.id + '" title="' + t.options.sourcechooserText + '" aria-label="' + t.options.sourcechooserText + '"></button>'+
+						'<div class="mejs-sourcechooser-selector mejs-offscreen" role="menu" aria-expanded="false" aria-hidden="true">'+
+							'<ul>'+
+							'</ul>'+
+						'</div>'+
+					'</div>')
 					.appendTo(controls)
 
 					// hover
 					.hover(function() {
 						clearTimeout(hoverTimeout);
-						$(this).find('.mejs-sourcechooser-selector').removeClass('mejs-offscreen');
+						$(this).find('.mejs-sourcechooser-selector')
+							.removeClass('mejs-offscreen')
+							.attr('aria-expanded', 'true')
+							.attr('aria-hidden', 'false');
 					}, function() {
 						var self = $(this);
 						hoverTimeout = setTimeout(function () {
-							self.find('.mejs-sourcechooser-selector').addClass('mejs-offscreen');
+							self.find('.mejs-sourcechooser-selector')
+							.removeClass('mejs-offscreen')
```

Comment:
this should be the opposite of the onhover function, and was the result of a bad merge. fix incoming.